### ISSUE TITLE
CSS and SCSS files very long

### ISSUE THREAD

**Issue by RhodesPeter** [The-Authenticators/sohapp#52]
At 262 and 336 lines long your CSS and SCSS files are a little on the long side. Would there be a way to modularise these?

**Comment by marisid** [comment]
Yes, that's a good point Peter. We originally thought about that but didn't have time to split. We can 'modularize' today.

**Comment by stevehopkinson** [comment]
We basically set up SCSS then wrote a load of vanilla CSS in it. #yolo

**Comment by Jwhiles** [comment]
presumably we could have written a number of scss files, and imported them all into some sort of index?
